cablelabs/snaps-kubernetes
# Helm issue 19

#### What does this PR do?
Fixes #19 
Adds Helm flag to configuration which tells Kubespray to install the 'helm' exe on the master(s) and deploy the 'tiller' service.
#### Do you have any concerns with this PR?
No
#### How can the reviewer verify this PR?
Ensure that the 'helm' executable has been installed on a master and ensure that the 'tiller' service is running
#### Any background context you want to provide?
DO NOT MERGE until the 'CI-ubunut18-validation' branch has been merged
Pulling low-hanging fruit from backlog
#### Screenshots or logs (if appropriate)
#### Questions:
- Have you connected this PR to the issue it resolves?
yes
- Does the documentation need an update?
Completed
- Does this add new Python dependencies?
No
- Have you added unit or functional tests for this PR?
Yes, added validation to ensure that the 'tiller' service is running in K8s
- Does this patch update any configuration files?
no
